src/policies/conditions.ts

```diff
@@ -118,6 +118,7 @@ export class Condition {
 
   protected makeReturnValueTest() {
     return Joi.object({
+      index: Joi.number().required(),
```

Comment:
Index is not a required parameter...https://github.com/nucypher/nucypher/pull/3032/files#diff-39ed7645b1d8386d5199d3de0d4bca1d523549567147e8c8c3f4f685b1f14da3R90